src/libmongoc/doc/learn/get/from-source.rst

```diff
@@ -0,0 +1,376 @@
+  **AlmaLinux**, etc.)::
+
+    # dnf install openssl-devel cyrus-sasl-devel
+
+  Package names may vary between distributions.
+
+
+.. _learn.get.configure:
+
+Configuring for |libbson|
+*************************
+
+.. important::
+
+  If you are building with Xcode [#xcode_env]_ or Visual Studio [#vs_env]_, you
+  may need to execute CMake from within a special environment in which the
+  resepective toolchain is available.
+
+Let the name `$BUILD` be the path `$SOURCE/_build`. This will be the directory
+where our built files will be written by CMake.
+
+With the source directory for |mongo-c-driver| at `$SOURCE` and build directory
+`$BUILD`, the following command can be executed from a command-line to configure
+the project with both |libbson| and |libmongoc|::
+
+  $ cmake -S $SOURCE -B $BUILD \
+    -D ENABLE_EXTRA_ALIGNMENT=FALSE \
+    -D ENABLE_AUTOMATIC_INIT_AND_CLEANUP=FALSE \
+    -D CMAKE_BUILD_TYPE=RelWithDebInfo \
+    -D ENABLE_MONGOC=FALSE
+
+
+If all dependencies are satisfied, the above command should succeed and end
+with::
+
+  $ cmake …
+  ## … (Lines of output) …
+  -- Generating done
+  -- Build files have been written to: $BUILD
+
+If configuration failed with an error, refer to the CMake output for error
+messages and information. Ensure that configuration succeeds before proceeding.
+
+.. dropdown:: Aside: What do these CMake arguments mean?
+  :class-container: admonition hint
+  :class-title: admonition-title
+  :animate: fade-in
+
+  The `ENABLE_EXTRA_ALIGNMENT` and `ENABLE_AUTOMATIC_INIT_AND_CLEANUP` are part
+  of |mongo-c-driver|, and correspond to deprecated features that are only
+  enabled for compatibility purposes. It is highly recommended to disable these
+  features whenever possible.
+
+  The `ENABLE_MONGOC=FALSE` argument disabled building |libmongoc|. We'll build
+  that in the next section.
+
+  The |cmvar:CMAKE_BUILD_TYPE| setting tells CMake what variant of code will be
+  generated. In the case of `RelWithDebInfo`, optimized binaries will be
+  produced, but still include debug information. The |cmvar:CMAKE_BUILD_TYPE| has no
+  effect on Multi-Config generators (i.e. Visual Studio), which instead rely on
+  the `--config` option when building/installing.
+
+.. _CMAKE_BUILD_TYPE: https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html
+
+
+.. _learn.get.build:
+
+Building the Project
+********************
+
+After successfully configuring the project, the build can be executed by using
+CMake::
+
+  $ cmake --build $BUILD --config RelWithDebInfo --parallel
+
+If configured properly and all dependencies are satisfied, then the above
+command should proceed to compile and link the configured components. If the
+above command fails, then there is likely an error with your environment, or you
+are using an unsupported/untested platform. Refer to the build tool output for
+more information.
+
+.. dropdown:: Aside: The ``--config`` option
+  :class-container: admonition hint
```

Comment:
Either "note" or "tip" seem more appropriate for these dropdowns than "hint". Consider changing the hint admonitions to either "note" (for extra details) or "tip" (for non-important suggestions).

Reply:
I chose "hint" purely for the iconography that Furo inserts (a circled question mark "?"), to indicate "Here's the answer to a question that you might have."